pkg/pool-weighted/test/ManagedPool.test.ts

```diff
@@ -471,7 +471,7 @@ describe('ManagedPool', function () {
             const joinSwapResult = await queryExitSwap();
             const joinResult = await queryEquivalentExit();
 
-            // Tokens are leaving the Vault and so is represented as a negative value.
+            // Tokens are leaving the Vault and so are represented as a negative value.
```

Comment:
```suggestion
            // Tokens are leaving the Vault, so they're represented as negative values.
```
Still had a number mismatch; might as well completely fix it.